Sources/Apodini/Response/Empty.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Empfy.swift
+//  
+//
+//  Created by Paul Schmiedmayer on 5/26/21.
+//
+
+
+public struct Empty: Encodable {
```

Comment:
Is there a reason why we don’t use Void for this?

Reply:
Yes, we have considered `Void` back in the days when we introduced `Empty` but you can not extend `Void` to e.g. conform to `Encodable`:
```swift
extension Void: Encodable {}
```
Results in the following compiler error: Non-nominal type 'Void' cannot be extended